.packit.yaml

```diff
@@ -37,3 +38,21 @@ jobs:
       manual_trigger: true
       targets:
           - fedora-rawhide
+
+    # run Cockpit storage tests, see plans/ with `revdeps == yes`
+    - job: tests
+      identifier: revdeps
```

Comment:
> Also, I have one request for using an identifier key in the existing job which I hope makes sense.

I previously had "self" (but argh, I put it onto the COPR build, not the test :see_no_evil: ), but changing it to "local" is fine for me (sounds a bit strange, though).

`adjust+` means that it adds to other `adjust+` rules, instead of overwriting it. I'm not sure how useful that is. I dropped the pluses for now, I doubt that it'll get significantly more complex anyway.

I also made the context selector more symmetrical, and it should now easily allow adding more test plans for other projects.

Reply:
To make it completely consistent, the identifier should actually be "all" now, to correspond to plans/all.fmf. Please let me know if you want me to rename the file to "local.fmf", or change the identifier to "all".

Reply:
Ah, the explicit `plan = all` [fails](https://artifacts.dev.testing-farm.io/b72bbe06-4a86-4b19-9083-cf164ce593c2/), and I also think that will badly break downstream gating. I reverted that.